salt/_modules/metalk8s_network.py

```diff
@@ -3,6 +3,10 @@
 import itertools
 import logging
 
+# Note: psutil is a dependency of Salt RPMs in MetalK8s context we
```

Comment:
Would it make sense to still explicitly install the package(s) our custom modules/states/.... depend on when we install the Salt package in some script(s), even when they're dependencies of said Salt packages and would hence be installed implicitly?

Reply:
Hmm sure we can, make sense I will add it

Reply:
Done